stacks/utils/getFunctionProps.ts

```diff
@@ -42,13 +44,24 @@ export const getEventBusFunctionProps = () => {
   };
 };
 
-export const getFunctionProps = () => {
+export const getFunctionProps = (stack: Stack) => {
   const { scheduleHandlerLambda, schedulerFunctionInvokeRole } =
     use(SchedulerStack);
 
   const dbFunctionProps = getDbFunctionProps();
   const eventBusFunctionProps = getEventBusFunctionProps();
 
```

Comment:
Im not sure that we should include resource creation into this util function.

Reply:
I wanted to put it into the scheduler stack but it threw a circular dependency error (tbh no idea why but couldn't get it ti work)

I put it here because it's the place for common functionalities between lambdas and this is one of them.